reconciler/events_test.go

```diff
@@ -0,0 +1,107 @@
+/*
+Copyright 2019 The Knative Authors
+
+Licensed under the Apache License, Veroute.on 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package reconciler
+
+import (
+	"errors"
+	"io"
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+	corev1 "k8s.io/api/core/v1"
+)
+
+const (
+	exampleStatusFailed = "ExampleStatusFailed"
+)
+
+func TestNil_Is(t *testing.T) {
+	var err error
+	if EventIs(err, NewEvent(corev1.EventTypeWarning, exampleStatusFailed, "")) {
+		t.Error("did not expect error to be a ReconcilerEvent")
```

Comment:
```suggestion
		t.Error("Did not expect error to be a ReconcilerEvent")
```

Reply:
you don't capital errors.

Reply:
But you capitalize logs :)

Reply:
You do capital errors in the unit tests. Those are _not_ errors.